test/uk/gov/hmrc/timetopayproxy/connectors/TtpFeedbackLoopConnectorSpec.scala

```diff
@@ -274,6 +279,63 @@ final class TtpFeedbackLoopConnectorSpec
             )
         }
       }
+
+      "using InternalAuth" should {
+        "return a successful response" in new Setup(ifImpl = false, internalAuthEnabled = true) {
+          stubPostWithResponseBodyEnsuringRequest(
+            "/debts/time-to-pay/cancel",
+            Json.toJson(ttpCancelRequest).toString(),
+            200,
+            Json.toJson(ttpCancelResponse).toString()
+          )
+
+          val result = connector.cancelTtp(ttpCancelRequest)
+
+          await(result.value) mustBe Right(ttpCancelResponse: TtpCancelSuccessfulResponse)
+        }
+
+        "parse an error response from an upstream service" in new Setup(ifImpl = false, internalAuthEnabled = true) {
+          stubPostWithResponseBodyEnsuringRequest(
```

Comment:
Do you need to add a test for when we receive unauthorised response and internal auth is enabled?